demo/radar.html

```diff
@@ -225,6 +232,14 @@
       // viewer.scene.view.lookAt(new THREE.Vector3(299900.954, 4701576.919, 66.197));
 			updateSidebar(visualizationMode);
 
+			window.calibrationModeAvailable = calibrationModeAvailable;
+			if (window.calibrationModeAvailable) {
+				document.getElementById("toggle_calibration_panels").style.display = "block";
+			} else {
+				document.getElementById("toggle_calibration_panels").style.display = "none";
```

Comment:
Use conditional expression to simplify.
`document.getElementById("toggle_calibration_panels").style.display = window.calibrationModeAvailable ? "block" : "none";`

Reply:
Done